app/routes/_index.tsx

```diff
@@ -1,22 +1,84 @@
-import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import {
+    Breadcrumb,
+    BreadcrumbBar,
+    BreadcrumbLink,
+    Card,
+    CardGroup,
+    CardHeader,
+    Header,
+    Label,
+    Link,
+    Select,
+} from "@trussworks/react-uswds";
+import React from "react";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
-        <CardGroup>
-            <Card>
-                <CardHeader>
-                    <h1 className="usa-card__heading">Meeting Spaces</h1>
-                    <p className="font-sans text-base-darker">
-                        Austin Public Library Meeting Spaces are
-                        <strong className="font-bold"> free of charge</strong> and ideal for
-                        discussion groups, panels, and lectures. Both paper and online reservation
-                        requests are timestamped and processed in the order they are received.
-                    </p>
-                </CardHeader>
-            </Card>
-        </CardGroup>
+        <div className="flex justify-center">
+            <CardGroup className="min-w-[30rem] max-w-[62rem]">
```

Comment:
Do these arbitrary width values match the specifications in Figma?

Reply:
Updated the max-width to be more aligned with the Figma spec; I changed it to `max-w-[49rem]`, which is 784 px, so not *quite* to spec (which is 790px). Should I use `px` over `rem` at this point? Also, should I be concerned with responsiveness?

Reply:
Using `rem` is correct. I'm guessing Jerri just used approximate values and didn't follow a strong grid since her designs are more in between wireframes and mock-ups.